.github/workflows/sival_nightly.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Nigthly
+on:
+  schedule:
+    - cron: "00 04 * * *"
+
+jobs:
+  FPGA-CW310-SiVal-Nightly:
+    name: FPGA CW310 SiVal nightly tests
+    runs-on: [ubuntu-20.04-fpga, cw310]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Required for the bitstream cache work.
+          ref: earlgrey_1.0.0 # schedule only work on the default branch, but we want to run another branch.
+
+      - uses: ./.github/actions/install-deps
+      - name: ROM boot stage
+        run: |
+          module load xilinx/vivado
+          ./bazelisk.sh sync --configure
+          ./bazelisk.sh run //sw/host/opentitantool -- --interface=hyperdebug_dfu transport update-firmware
+          ./bazelisk.sh test --build_tests_only \
+          $(bash ./bazelisk.sh query 'attr("tags", "[\[ ]cw310_sival[,\]]", tests(//...))' \
```

Comment:
I think we should do this as a "bazel test pattern file" in case the list of tests gets too long to fit in command line arguments.

It would be something like

```sh
bazel_tests="$(mktemp)"
./bazelisk.sh query 'attr("tags", "[\[ ]cw310_sival[,\]]", tests(//...))' \
    | grep -v examples \
    | grep -v third_party \
    | grep -v penetrationtests \
    > "$bazel_tests"
./bazelisk.sh test --build_tests_only --target_pattern_file="$bazel_tests"
```